test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```

Comment:
test file, tmp file, public/system, public/assets you can define in gitignor
dont need do test

Reply:
![image](https://user-images.githubusercontent.com/68467813/89922316-730d9680-dc31-11ea-8fd6-2ca0d3fa52c4.png)

What should i do to solve this ques?

Reply:
is that /tmp correct as comment in gitignore for ignore tmp file>

Reply:
is that /tmp correct as comment in gitignore for ignore tmp file>